<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pelu_Dialog</class>
 <widget class="QDialog" name="Pelu_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Peluquería</string>
  </property>
  <widget class="QLabel" name="lbl_tratamientos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tratamientos disponibles:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_tratamientos">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <item>
    <property name="text">
     <string>Corte 10€</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Peinado 10€</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tinte 25€</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mechas 30€</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Alisado 50€</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbl_resultados">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_seleccion">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tratamiento seleccionado: </string>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_trat_seleccionados">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_agregar_tto">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Tratamiento: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_agregar_tto">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_agregar_tto">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hl_modificacion_lista">
    <item>
     <widget class="QPushButton" name="btn_editar">
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_eliminar">
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_eliminar_todos">
      <property name="text">
       <string>Eliminar Todos</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
